code/Data exploration.R

```diff
@@ -53,8 +56,37 @@ data <- data %>% arrange(DATETIME);
 ##fill in missing temperature (down fill)
 data <- data %>% fill(TEMPERATURE)
 
-##convert DATETIME to date
-data$DATETIME = ymd_hms(data$DATETIME);
+
+##create table with period of initial forecast
+initial_forecast <- forecastdemand_nsw %>% 
+  filter(PERIODID==1) %>% 
```

Comment:
what does period mean here?

Reply:
AEMO make multiple estimates between the periods, with the subsequent prediction made based on the previous. I'm just grabbing the initial and final predictions in case if they're needed.